### DISCUSSION TITLE
Plex not playing well with symlinks created by RDTclient in Windows

### DISCUSSION THREAD

**Discussion by cculpepp75** [rogerfar/rdt-client#473] [General]
Running rdt client in Windows as a service with Zurg and the arrs.  Everything is set up and running perfectly except for the fact that Plex does not recognize the symlinks created by RDTclient when it scans the media folder.  Its sees a blank folder and scans nothing into Plex. The symlinks are there, and I can even open and play them directly from the file, but Plex isn't seeing them. Any ideas?  ETA...everything is running as a service in Windows 10...no docker.

**Comment by rogerfar** [comment]
This seems more like a Plex issue, not sure if Plex even recognizes Windows symlink files?

**Comment by MrJoshGit** [comment]
I am having same problem as OP.  I will do some more research.  
Jellyfin is able to read/scan the library and symlinks work fine with it.
